WeakAuras/Prototypes.lua

```diff
@@ -11224,18 +11224,62 @@ Private.event_prototypes = {
       ["events"] = {
         "CHAT_MSG_CURRENCY",
         "CURRENCY_DISPLAY_UPDATE",
+        "ACCOUNT_CHARACTER_CURRENCY_DATA_RECEIVED",
+        "CURRENCY_TRANSFER_LOG_UPDATE",
```

Comment:
i was expecting this to error on cataclysm, but it didn't

Reply:
non-retail flavors don't have `currencyInfo.isAccountTransferable` which is what is gatekeeping any retail code from running on cata.